sema/syncfs/service.py

```
@@ -186,15 +204,30 @@ def __init__(
-            from_path=self.source_path,
-            to_store=self.rdfstore,
-        )
+        try:
+            if self.rdfstore:
+                perform_sync(
+                    from_path=self.source_path,
+                    to_store=self.rdfstore,
+                )
+                self._result.success = True
+        except FileNotFoundError as e:
+            log.error("Source file not found during sync", exc_info=e)
```

Ruff

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)